### ISSUE TITLE
Zotereo metadata - empty entry

### ISSUE THREAD

**Issue by sladen** [ShadowItaly/clas-digital#266]
* https://www.clas-digital.uni-frankfurt.de/books/3P7C5HKJ/

is an empty entry.  Ideally investigate what is going on.

**Comment by ekoehring** [comment]
Because of the usability of the Zotero Client, it is very easy to accidentally create empty entries (just consisting of ItemType), there is no confirmation question for that. When I spot them, I delete them.

You could help by making some sort of filter (if all other fields empty --> do not show), but mainly it is a zotero problem.

**Comment by ekoehring** [comment]
(Not deleting this one, so that you can play with it - let meknow if you dont need it)

**Comment by georgbuechner** [comment]
Shall we implement I small check, whether a few basic entries are set?

**Comment by ekoehring** [comment]
Yes, that was my idea: if all fields apart from item type are empty --> do not show
If that is easy to do, it could be nice.

**Comment by georgbuechner** [comment]
Just one more questions concerning this: I have an implementation, that when "author", "title", "date", and "place" are not set, then we ignore the book entirely. Shall we list the books, so it will be easier for you to delete the zotero entries?

**Comment by georgbuechner** [comment]
So, I uploaded a small fix. But running it on the development server indicates that there currently are no books __in the catalogue__ with no entries. Also you can run a search in the catalogue for the book mention by sladen, however you can't find it. I guess this is some sort of depricated data. Which needs further investigation.
